content/events/online-hackfest/2020-uiux/index.adoc

```diff
@@ -102,11 +101,45 @@ Advanced topics:
 
 == Online events
 
-We are going to organize a number of online sessions during the hackfest to
-share experiences and do knowledge transfers (Jenkins UI and documentation development).
-There will be an opening event on May 25.
-Stay tuned, they will be announced soon!
-
+We will organize multiple online sessions during the hackfest to
+share experiences and do knowledge transfers.
+All these sessions will be recorded and published on link:https://www.youtube.com/user/jenkinsci/playlists[our YouTube channel].
+There are no mandatory events you must attend remotely, you are welcome to join the events you are interested in.
+
+// TODO: Add timezone converter for all dates
+
+* May 25, 01:00-02:30 PM UTC -
+  _Hackfest kick-off_
+  (link:https://www.meetup.com/Jenkins-online-meetup/events/270644129/[RSVP here])
+* May 25, 03:00-04:00 PM UTC -
+  _Beautify the user interface of Jenkins reporter plugins_ by link:/blog/authors/uhafner/[Ullrich Hafner]
+  (TBA)
+* May 25, 04:00-04:30 PM UTC -
+  _Jenkins is The Way program overview_ by link:/blog/authors/alyssat/[Alyssa Tong]
+  (TBA)
+* May 26, 09:00-10:00 AM UTC -
+  _Additional Opening and Q&A session_
+  (TBA)
+* May 26, 10:00-10:45 AM UTC -
+  _Migrating plugin docs to Documentation-as-Code_ by link:/blog/authors/oleg_nenashev/[Oleg Nenashev]
+  (TBA)
+* May 26, TBA -
+  _System Read Permission: Feature demo, and how to contribute_ by link:/blog/authors/timja/[Tim Jacomb]
```

Comment:
should really have an author macro for this

Reply:
Could you please report a GitHub issue?

Reply:
https://github.com/jenkins-infra/jenkins.io/issues/3261, tried to use the create an issue from the comment but it seems broken atm =/, annoying